mldsa/poly.c

```diff
@@ -174,9 +174,19 @@ void mld_poly_power2round(mld_poly *a1, mld_poly *a0, const mld_poly *a)
 
 void mld_poly_decompose(mld_poly *a1, mld_poly *a0, const mld_poly *a)
 {
+#if defined(MLD_USE_NATIVE_POLY_DECOMPOSE_88) && MLDSA_MODE == 2
+  /* TODO: proof */
+  mld_assert_bound(a->coeffs, MLDSA_N, 0, MLDSA_Q);
+  mld_poly_decompose_88_native(a1->coeffs, a0->coeffs, a->coeffs);
+#elif defined(MLD_USE_NATIVE_POLY_DECOMPOSE_32) && \
```

Comment:
Sorry for the dumb question, but why the name `decompose_32` and `decompose_88`?